frontend/src/app/footer/footer.component.html

```diff
@@ -1,5 +1,5 @@
-<footer class="footer">
-  <div class="container">
+<div class="container">
+  <footer class="footer">
```

Comment:
Please move .container to .footer, so we won't nest so deeply. Adjust paddings in .footer in css